frontend/benefit/handler/src/hooks/useUserQuery.ts

```diff
@@ -46,6 +46,12 @@ const useUserQuery = <T extends User>(
     }
   };
 
+  const onSuccessHandler = (user: User): void => {
+    checkForStaffStatus(user);
+    // eslint-disable-next-line scanjs-rules/identifier_localStorage
+    localStorage.setItem(LOCAL_STORAGE_KEYS.CSRF_TOKEN, user.csrf_token);
```

Comment:
These `// eslint-disable-next-line scanjs-rules/identifier_localStorage` start to get out of hand. We should make a `localStorage` **util** or maybe [useLocalStorage](https://www.npmjs.com/package/use-local-storage) sort of thing (don't know if this is THE lib to use).

Reply:
I'll make a new ticket out of this.

Reply:
Yeah true, I will do this too here